Fix broken CI build — TS2352 in governance-health.ts

## Problem

CI builds on `main` are broken. The `tsc -b` step fails with:

```
src/utils/governance-health.ts(182,10): error TS2352: Conversion of type 'ProposalPipelineCounts'
to type 'Record<string, number>' may be a mistake because neither type sufficiently overlaps with the other.
```

This affects all PRs (e.g., #200) — they show CI failures even when their own changes are correct.

## Root Cause

PR #189 added `extendedVoting` to the `ProposalPipelineCounts` interface, making it a proper typed field. However, `governance-health.ts` still uses a defensive pattern from before the field existed:

```ts
const extendedVoting =
  'extendedVoting' in pipeline
    ? (pipeline as Record<string, number>).extendedVoting
    : 0;
```

The `as Record<string, number>` cast now fails TypeScript's type narrowing because `ProposalPipelineCounts` has specific fields, not an index signature.

## Fix

Replace the defensive check with direct field access:

```ts
pipeline.extendedVoting
```

Since `extendedVoting` is now a declared field on `ProposalPipelineCounts`, the runtime check and unsafe cast are unnecessary.

## Impact

- All CI builds fail until this is fixed
- No behavioral change — the field has always been present in practice since PR #189

--- a/web/src/utils/governance-health.ts
+++ b/web/src/utils/governance-health.ts
@@ -175,15 +175,10 @@ export function computePipelineFlow(metrics: GovernanceMetrics): SubMetric {
   const { pipeline } = metrics;
   const total = pipeline.total;
 
-  // Proposals that advanced past discussion.
-  // Include extendedVoting if present (added by PR #189).
-  const extendedVoting =
-    'extendedVoting' in pipeline
-      ? (pipeline as Record<string, number>).extendedVoting
-      : 0;
+  // Proposals that advanced past discussion
   const advanced =
     pipeline.voting +
-    extendedVoting +
+    pipeline.extendedVoting +
     pipeline.readyToImplement +
     pipeline.implemented +
     pipeline.rejected +

